vagrant/Makefile

```diff
@@ -131,14 +130,6 @@ grafana-dashboards:
 	kubectl delete configmap -n "${namespace}" sumologic-dashboards || true
 	kubectl create configmap -n "${namespace}" sumologic-dashboards --from-file="${dashboards_path}"
 
-# Create secrets with microk8s certs
-create-secrets:
-	kubectl delete secret -n "${namespace}" microk8s-certs || true
-	kubectl create secret -n "${namespace}" generic microk8s-certs \
-		--from-file=/var/snap/microk8s/current/certs/server.key \
-		--from-file=/var/snap/microk8s/current/certs/server.crt \
-		--from-file=/var/snap/microk8s/current/certs/ca.crt
-
```

Comment:
Why do we remove this?

Reply:
It was used for service monitor before you changed the kubelet startup configuration. We do not use it anywhere

Reply:
🤷🏻